src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetCaseByIdTest.java

```diff
@@ -19,14 +24,76 @@ class CaseControllerGetCaseByIdTest extends IntegrationBase {
 
     private static String endpointUrl = "/cases/{caseId}";
 
+    private static final OffsetDateTime SOME_DATE_TIME = OffsetDateTime.parse("2023-01-01T12:00Z");
+    private static final String SOME_COURTHOUSE = "some-courthouse";
+    private static final String SOME_COURTROOM = "some-courtroom";
+    private static final String SOME_CASE_NUMBER = "1";
+
+    @BeforeEach
+    void setUp() {
+
+        dartsDatabase.givenTheDatabaseContainsCourtCaseWithHearingAndCourthouseWithRoom(
+            SOME_CASE_NUMBER,
+            SOME_COURTHOUSE,
+            SOME_COURTROOM,
+            SOME_DATE_TIME.toLocalDate()
+        );
+
+    }
+
     @Test
     void casesSearchGetEndpoint() throws Exception {
 
-        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, "25");
+        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, getCaseId());
+
+        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isOk());
 
-        mockMvc.perform(requestBuilder).andExpect(MockMvcResultMatchers.status().isNotImplemented());
+    }
+
+    @Test
+    void casesSearchGetEndpointCheckListsAreCorrectSize() throws Exception {
+
+        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, getCaseId());
+
+        mockMvc.perform(requestBuilder)
+            .andExpect(MockMvcResultMatchers.status().isOk())
+            .andExpect(MockMvcResultMatchers.jsonPath("$.case_id", Matchers.is(getCaseId())))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.judges", Matchers.hasSize(1)))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.prosecutors", Matchers.hasSize(1)))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.defendants", Matchers.hasSize(1)))
+            .andExpect(MockMvcResultMatchers.jsonPath("$.defenders", Matchers.hasSize(1)));
 
     }
 
+    @Test
+    void casesSearchGetEndpointCheckObject() throws Exception {
+
+        MockHttpServletRequestBuilder requestBuilder = get(endpointUrl, getCaseId());
+
+        mockMvc.perform(requestBuilder)
```

Comment:
Looks like this is the same test as the one above but just checking different items in the response. These checks could be combined into one test.